features/dd-sdk-android-flags/src/test/kotlin/com/datadog/android/flags/internal/DatadogFlagsClientTest.kt

```diff
@@ -97,7 +100,8 @@ internal class DatadogFlagsClientTest {
                 rumIntegrationEnabled = true
             ),
             rumEvaluationLogger = mockRumEvaluationLogger,
-            processor = mockProcessor
+            processor = mockProcessor,
+            flagStateChannel = FlagsStateChannel(DDCoreSubscription.create())
```

Comment:
it is better to introduce `mockFlagsStateChannel` here and verify `addListener`/`removeListener` against it instead of having only `verifyNoInteractions(mockListener)`.

Reply:
done.